### DISCUSSION TITLE
O template para Diário está bom?

### DISCUSSION THREAD

**Discussion by christinaflachufba** [mate28-ic-ufba/turma-20212#33] [General]
Veja o [template](https://github.com/mate28-ic-ufba/turma-20212/blob/main/diarios/template.md).
Ainda dá tempo de sugerir melhorias (até hoje, final do dia?)

**Comment by danielafeitosa** [comment]
Pode incluir também:
- O que foi feito e se teve problemas
- Qual o próximo passo e se precisa de ajuda
